vendor/assets/javascripts/behaviors/selectable_asset.js

```diff
@@ -0,0 +1,17 @@
+Cmsify.SelectableAsset = function(el) {
+  this.$el = $(el);
+  this.select = this.select.bind(this);
```

Comment:
Ahh, now I see, might be better to make it a single line like `this.$el.on('click', this.select.bind(this));`